perun/postprocess/regression_analysis/extensions/plot_models.py

```diff
@@ -0,0 +1,147 @@
+""" Extension for regression model coefficients transformation into array of points. The points
+    array can be then used for model plotting as a series of lines forming a curved line.
+"""
+
+import perun.postprocess.regression_analysis.tools as tools
+import numpy as np
+
+
+# Default model curve smoothness specified as number of points generated from x interval
+# The higher, the smoother are the curves
+DEFAULT_SMOOTHNESS = 51
```

Comment:
Minor Remark: Maybe explaining, why 51 was chosen constant would increase the readability and help others when adjusting the smootness.